dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h

```diff
@@ -25,6 +17,24 @@ void LogErrorInMemory(HRESULT hr, ULONG line, _In_ PCSTR filename, _In_ PCWSTR m
 #define LOG_ERROR_IN_MEMORY(hr, line, filename, message) __noop
 #endif
 
+class MrtRuntimeProvider : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(MrtRuntimeProvider, "Microsoft.Windows.Mrt.Runtime",
+        // {706f5ec1-0c6b-54ee-4b6f-4cd329da9067}
+        (0x706f5ec1, 0x0c6b, 0x54ee, 0x4b, 0x6f, 0x4c, 0xd3, 0x29, 0xda, 0x90, 0x67));
+
+    DEFINE_COMPLIANT_TELEMETRY_EVENT_PARAM3(TelemetryGenericEventParam1, PDT_ProductAndServicePerformance, PCWSTR, functionName, PCWSTR, message, int, hresult);
```

Comment:
Do you want these event names be to generic?

Reply:
Yes, it's intentional. I chose this based on what we had in our codebase already.